server/balancer_worker.go

```diff
@@ -203,3 +231,41 @@ func (bw *balancerWorker) doBalance() error {
 	log.Info("find no proper region for balance, retry later")
 	return nil
 }
+
+func (bw *balancerWorker) mockBalanceOperator() {
```

Comment:
It is convenient for us to have a test.
How about moving it to a new file called `mock.go`?